ui/src/app/workflows/components/workflow-details/workflow-details.tsx

```diff
@@ -176,7 +176,13 @@ export const WorkflowDetails = ({history, location, match}: RouteComponentProps<
         const retryWatch = new RetryWatch<Workflow>(
             () => services.workflows.watch({name, namespace}),
             () => setError(null),
-            e => setWorkflow(e.object),
+            e => {
```

Comment:
When the workflow is deleted, we get a message and we still show the workflow itself:

![image](https://user-images.githubusercontent.com/1142830/105527465-99336a00-5c98-11eb-9c29-e2385ce28f98.png)